src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -496,6 +496,10 @@ def _forward(self, model_inputs, return_timestamps=False, generate_kwargs=None):
             else:
                 generate_kwargs["encoder_outputs"] = encoder(inputs, attention_mask=attention_mask)
 
+            # If the tokenizer has a pad token but the model doesn't, we add it to the `generate` call
+            if self.tokenizer.pad_token_id is not None and self.model.generation_config.pad_token_id is None:
```

Comment:
I'm assuming we can assume `generation_config` will have the `pad_token_id` attribute?

Reply:
Yes :) it is part of [`__init__`](https://github.com/huggingface/transformers/blob/6b660d5ed561cc7326e77e3d833cde8085b23663/src/transformers/generation/configuration_utils.py#L353)